app/controllers/recipe_ingredients_controller.rb

```diff
@@ -0,0 +1,10 @@
+class RecipeIngredientsController < ApplicationController
+  def update
+    redirect = Recipe.find(params[:id])
+    recipe = Recipe.create!(name: params[:name], cost: params[:cost])
```

Comment:
The 2nd extension asks us to add an existing ingredient to an existing recipe. As such, we don't need to create a new `Recipe` object, just a new relationship between those two existing objects. So, we really only need to `.find` the recipe by its ID, so that we can relate an ingredient and this recipe together.